pulser-core/pulser/backend/remote.py

```diff
@@ -70,6 +70,17 @@ def results(self) -> tuple[Result, ...]:
         """The actual results, obtained after execution is done."""
         return self._results
 
+    @property
+    def submission_id(self) -> str:
```

Comment:
Reminder that we still need to decide whether we want to use "batch" instead of "submission"